internal/server/get_payments.go

```diff
@@ -76,11 +78,22 @@ func GetPayments(client GetPaymentsClient, tmpl template.Template) Handler {
 			return err
 		}
 
-		total := 0
+		totalPaid := 0
+		totalPaidAndReductions := 0
 		for _, p := range payments {
-			total = total + p.Amount
+			if p.Source != sirius.FeeReductionSource {
+				totalPaid = totalPaid + p.Amount
+			}
+			totalPaidAndReductions = totalPaidAndReductions + p.Amount
+		}
+		data.TotalPaid = totalPaid
```

Comment:
Rather than create a separate variable, you can just add to `data.TotalPaid` directly (it initialises as `0`)
```suggestion
			if p.Source != sirius.FeeReductionSource {
				data.TotalPaid = data.TotalPaid + p.Amount
			}
			totalPaidAndReductions = totalPaidAndReductions + p.Amount
		}
```